spec/support/geocoder_default_location.rb

```diff
@@ -38,7 +39,7 @@
   let(:bounding_box) { [39.989124784445764, -74.96065051723293, 41.43644261555424, -73.05123208276707] }
 
   before do
-    Geocoder.configure(lookup: :test)
+    Geocoder.configure(lookup: :test, ip_lookup: :test)
```

Comment:
Oooh. Nifty, I didn't know about this. This might fix some broken/untested things.